src/rules/declaration-nested-properties/README.md

```diff
@@ -0,0 +1,144 @@
+# declaration-nested-properties
+
+Require or disallow properties with `-` in their names to be in a form of a nested group.
+
+```scss
+/* This is properties nesting: */  
+font: {
+  size: 16px;
+  weight: 700;
+}
+```
+
+[Sass official docs on nested properties](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#nested_properties).
+
+## Options
+
+`string`: `"always"|"never"`
+
+### `"always"`
+
+*Every property* with a `-` in its name *must be* inside a nested property group.
+
+The following patterns are considered warnings:
+
+```scss
+a {
+  margin-left: 10px;
+}
+```
+
+```scss
+a {
+  font: {
+    size: 10px;
+  }
+  font-weight: 400; // This one should be nested either
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```scss
+a {
+  margin: {
+    left: 10px;
+  }
+}
+```
+
+```scss
+a {
+  font: 10px/1.1 Arial {
+    weight: bold;
+  }
+}
+```
+
+### `"never"`
+
+Nested properties are not allowed.
+
+The following patterns are considered warnings:
+
+```scss
+a {
+  margin: {
+    left: 10px;
+  }
+}
+```
+
+```scss
+a {
+  background: red {
+    repeat: no-repeat;
+  }
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```scss
+a {
+  background-color: red;
+  background-repeat: no-repeat;
+}
+```
+
+```scss
+a {
+  background:red {
+    color: blue;
+  }
+}
+
+/* There is no space after the colon in `background:red` so it is considered A SELECTOR and is compiled into: 
+
+a background:red {
+  color: blue;
+}
+
+*/
+```
+
+## Optional options
+
+### `except: ["only-of-namespace"]`
```

Comment:
I think that the option would be easier to understand if it was called `"only-one-of-namespace"` instead of `"only-of-namespace"`. What do you think?